KnockKnock-iOS/Scenes/Challenge/ChallengeViewController.swift

```diff
@@ -24,37 +24,71 @@ final class ChallengeViewController: BaseViewController<ChallengeView> {
   }
-  
+
+  override func viewWillAppear(_ animated: Bool) {
+    self.tabBarController?.tabBar.isHidden = false
+  }
+
   override func setupConfigure() {
-    self.containerView.tableView.do {
+    self.containerView.challengeCollectionView.do {
       $0.delegate = self
       $0.dataSource = self
     }
+    self.containerView.sortChallengeButton.addTarget(self, action: #selector(tapSortChallengeButton(_:)), for: .touchUpInside)
+  }
+
+  func setNavigationItem() {
+    let searchBarButtonItem = UIBarButtonItem(barButtonSystemItem: .search, target: self, action: nil)
+
+    self.navigationItem.title = "챌린지"
+    self.navigationItem.rightBarButtonItem = searchBarButtonItem
+  }
+
+  @objc func tapSortChallengeButton(_ sender: UIButton) {
+    let bottomSheetViewController = BottomSheetViewController()
+    bottomSheetViewController.modalPresentationStyle = .overFullScreen
+    self.present(bottomSheetViewController, animated: false, completion: nil)
   }
 }
 
 extension ChallengeViewController: ChallengeViewProtocol {
   func fetchChallenges(challenges: [Challenge]) {
     self.challenges = challenges
-    self.containerView.tableView.reloadData()
+    self.containerView.challengeCollectionView.reloadData()
   }
 }
 
-extension ChallengeViewController: UITableViewDataSource {
-  func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+extension ChallengeViewController: UICollectionViewDataSource {
+  func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
     return self.challenges.count
   }
-  
-  func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-    let cell = tableView.dequeueCell(withType: ChallengeCell.self, for: indexPath)
+
+  func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+    let cell = collectionView.dequeueReusableCell(withReuseIdentifier: "cell", for: indexPath) as! ChallengeCell
     let challenge = self.challenges[indexPath.row]
+    cell.backgroundColor = .white
     cell.bind(data: challenge)
     return cell
   }
 }
 
-extension ChallengeViewController: UITableViewDelegate {
-  
+extension ChallengeViewController: UICollectionViewDelegateFlowLayout {
+  func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, sizeForItemAt indexPath: IndexPath) -> CGSize {
+    return CGSize(width: (self.containerView.frame.width - 40), height: (self.containerView.frame.height - 120) / 2)
+  }
+  func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, minimumLineSpacingForSectionAt section: Int) -> CGFloat {
+    return 40
+  }
+  func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, minimumInteritemSpacingForSectionAt section: Int) -> CGFloat {
+    return 20
+  }
```

Comment:
```suggestion
  func collectionView(
    _ collectionView: UICollectionView, 
    layout collectionViewLayout: UICollectionViewLayout,
     sizeForItemAt indexPath: IndexPath
   ) -> CGSize {
      return CGSize(width: (self.containerView.frame.width - 40), height: (self.containerView.frame.height - 120) / 2)
  }
  
  func collectionView(
    _ collectionView: UICollectionView,
    layout collectionViewLayout: UICollectionViewLayout,
    minimumLineSpacingForSectionAt section: Int
  ) -> CGFloat {
      return 40
  }
  
  func collectionView(
    _ collectionView: UICollectionView,
    layout collectionViewLayout: UICollectionViewLayout,
    minimumInteritemSpacingForSectionAt section: Int
  ) -> CGFloat {
    return 20
  }
```

이렇게 변경하는게 저희 컨벤션에 맞아 보입니다.
[스타일쉐어 스타일가이드](https://github.com/StyleShare/swift-style-guide#%EC%B5%9C%EB%8C%80-%EC%A4%84-%EA%B8%B8%EC%9D%B4) 에서 xcode 설정을 저희도 한 줄 최대 길이를 99자로 설정하시죠 👍